PhpAmqpLib/Channel/AMQPChannel.php

```diff
@@ -1261,10 +1255,10 @@ protected function basic_return($args, $msg)
                 $routing_key,
                 $msg,
             ));
-
-        } else {
-            $this->debug->debug_msg('Skipping unhandled basic_return message');
+            return;
         }
+
+        $this->debug->debug_msg('Skipping unhandled basic_return message');
```

Comment:
Do you think we could reverse the `if`?  I think it would be more clear if the happy-path was straight down, and the skipping path was in the `if`.